packages/electron-app/src/main/index.ts

```diff
@@ -4,21 +4,72 @@
 
 import electron from 'electron';
 import installExtension, { REACT_DEVELOPER_TOOLS } from 'electron-devtools-installer';
-import Pino from 'pino';
 import path from 'path';
+import Pino from 'pino';
 import url from 'url';
 
-import { CSP, staticPath } from './util';
+import { CSP, isSubstrateRunning, runSubstrateDev, staticPath } from './util';
 
 const { app, BrowserWindow, session } = electron;
-const pino = Pino();
-let mainWindow: Electron.BrowserWindow | undefined;
+const pino = new Pino();
+let sluiApp: Electron.BrowserWindow | undefined;
+let substratePid: number;
+let hasCalledInitParitySubstrate = false;
+
+pino.info('Platform detected: ', process.platform);
+pino.info('Process type: ', process.type);
+pino.info('Process ID: ', process.pid);
+pino.info('Process args: ', process.argv);
+pino.info('Electron version: ', process.versions['electron']);
+
+app.once('ready', async () => {
+  const setPid = (pid: number) => {
```

Comment:
imo childprocess.kill looks more idiomatic, any reason you prefer to work with pids?

Reply:
how would that work?

Reply:
https://nodejs.org/api/child_process.html#child_process_subprocess_kill_signal